app/domain/authentication/authn_jwt/configuration_interface.rb

```diff
@@ -1,8 +1,8 @@
 module Authentication
   module AuthnJwt
     # Interface containing JWT configuration functions to be implemented in JWTConfiguration class for a vendor
-    class JWTConfigurationInterface
-      def self.conjur_id; end
+    class ConfigurationInterface
+      def self.jwt_identity(authentication_parameters); end
```

Comment:
When you need to change an interface this might be red light of something not fully planned.
The design should have exactly defines the interface that is needed to implement it.
If it wasn't, please take the time to finalize the design with all interfaces, update the design doc, and change it aligned to the design.

I'm saying this b/c I don't want to review interface changes over and over. That's why we do design prior to writing code.

Reply:
The design was in higer level then specifying which parameters are passed between the components. So the things being passed were open to the the developers to decide because they are things that are more clear during development. (this about line 5).
line 4 is rename
The design in finished and agreed what @nessiLahav done

Reply:
@tzheleznyak Please update all functions to get the `authentication_parameters` as input